### PR TITLE
Removed Latest Version Check Help

### PR DESCRIPTION
**Pull request by ceford**
Pull Request for Issue # .

### Summary of Changes

Remove Latest Version Check Help screen from the Joomla! Help page in Joomla 4. 

### Testing Instructions

In Joomla 3 or Joomla 4 go to Help / Joomla! Help. The second item in the index is Latest Version Check. In Joomla 3 there is a warning if the installed version is not the current version and then a list of Announcements and Release notes going back through all major/minor/patch versions of the 3.x series. They link to the Release News pages. There are also Frequently Asked Questions for each major/minor/patch release. They are actually bug reports, are sparce and don't work properly when delivered by the Help proxy server. All of this has accumulated well over 400 Help support files. The Version Check is redundant because the Home dashboard does that check anyway in Joomla 4. 

In Joomla 4 the Latest Version Check Help link returns a 404 not found page because there are no Help support pages (yet).

This PR removes that Help page and the code required to generate it - all in com_admin.

### Actual result BEFORE applying this Pull Request

The Latest Version Check page is present in the Joomla! Help index for Joomla 4 but returns an error page.

### Expected result AFTER applying this Pull Request

The Latest Version Check page is absent from the Joomla! Help index for Joomla 4.

### Documentation Changes Required

None

### CODE REVIEW

**Comment by brianteeman** [comment]
The difference between the update check on the dashboard and the help page is that as you identified the help page includes
> in Joomla 3 there is a warning if the installed version is not the current version and then a list of Announcements and Release notes going back through all major/minor/patch versions of the 3.x series. 

I dont see a good reason for removing this link - or am I missing something

**Comment by ceford** [comment]
For Joomla 3 have a look at these two pages:
https://help.joomla.org/proxy/index.php?keyref=Help39:Joomla_Version_3_9_21/en&lang=en
https://help.joomla.org/proxy/index.php?keyref=Help39:Joomla_Version_3_9_20/en&lang=en
They are identical. So are all of the others through the the 3 series, hundreds of them. Except that of the current release which does not have the Warning message.
Have a look at this:
https://docs.joomla.org/Category:Version_3.9.22_FAQ
And compare it with this:
https://help.joomla.org/proxy/index.php?keyref=Category:Version_3.9.22_FAQ
The second version does not show the reported bug. And they are rare anyway.
If I try to fill out the form to create a FAQ I get taken here:
https://docs.joomla.org/Main_Page
Whatever we are trying to do we are doing it wrong!
It would be nice to have a page with abstracts of Release Announcements - but just one page. Separate issue?
And Bugs should be Bugs and not FAQS.

**Comment by brianteeman** [comment]
Everything you describe is accurate but the solution is not to remove the link here. The solution to all of them lies with the docs site

**Comment by MartijnMaandag** [comment]
I am sorry, the label says: "Latest version check"
I agree with Cliff that it is a duplicate of the automated version check in Joomla itself. If you don't see the automated one, you will never find this one in the helpscreens.
I always wondered why I did always have to translate these new release-notes and did not do it the latest 3.9 release. I do see now where it is used and that, if you don't translate the latest version, the hole is in Englisch and not translated.
So this is not a good version check and we must remove it from J4. If there is a need for additional version information, we must add this and create something on the docs to support this.

File: administrator/components/com_admin/tmpl/help/default.php [hunk 1]
```diff
@@ -23,7 +23,6 @@
 			<div class="sidebar-nav">
 				<ul class="nav flex-column">
 					<li><?php echo HTMLHelper::_('link', Help::createUrl('JHELP_START_HERE'), Text::_('COM_ADMIN_START_HERE'), ['target' => 'helpFrame']); ?></li>
-					<li><?php echo HTMLHelper::_('link', $this->latestVersionCheck, Text::_('COM_ADMIN_LATEST_VERSION_CHECK'), ['target' => 'helpFrame']); ?></li>
```
Comment (Quy) [inline administrator/components/com_admin/tmpl/help/default.php:26]: Please delete language string.

Comment (ceford) [reply administrator/components/com_admin/tmpl/help/default.php:26]: Removed language string as requested.

**Comment by tushar33** [comment]
I have tested this item :white_check_mark: successfully on e63d29f3a3e6abd1fb1aef3ea50a96ce1c60feca<br /><br />Showing expected result<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31038">issues.joomla.org/tracker/joomla-cms/31038</a>.</sub>

**Comment by karo3** [comment]
I have tested this item :red_circle: unsuccessfully on e63d29f3a3e6abd1fb1aef3ea50a96ce1c60feca<br /><br />The link will be removed, so that works. But the div container with the class row lacks the spacing class mt-sm-3. The PR must therefore be corrected at this point.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31038">issues.joomla.org/tracker/joomla-cms/31038</a>.</sub>

**Comment by Giuse69** [comment]
yes, just modify line 21 in administrator/components/com_admin/tmpl/help/default.php
from
`<div class="row">`
to
`<div class="row mt-sm-3">`
but strangely that line is not marked as modified in the "files changed" tab...why?

**Comment by ceford** [comment]
I don't understand why the php72-unit test is failing. Have I done something wrong? I did start a local build accidentally and cancelled it before it went very far.

**Comment by HLeithner** [comment]
I restarted drone

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on d158b2d79feec5c219c18cc4a816ba26bb4c20d2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31038">issues.joomla.org/tracker/joomla-cms/31038</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on d158b2d79feec5c219c18cc4a816ba26bb4c20d2<br /><br />The PR does what it claims to do, and all these links to release note pages are not very helpful anyway, and the version check we have elsewhere.

So for me it's ok to remove that page.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31038">issues.joomla.org/tracker/joomla-cms/31038</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31038">issues.joomla.org/tracker/joomla-cms/31038</a>.</sub>

**Pull request merged**